doc/radosgw/dynamicresharding.rst

```diff
@@ -0,0 +1,82 @@
+===================================
+RGW Dynamic Bucket Index Resharding
+===================================
+
+.. versionadded:: Luminous
+
+A large bucket index can lead to performance problems. In order
+to address this problem we introduced bucket index sharding.
+Until Luminous, changing the number of bucket shards (resharding)
+needed to be done offline, from Luminous we support
+online bucket resharding.
+
```

Comment:
can you add a note here mentioning not to use this in multisite scenarios

Reply:
right, will add it